Makefile

```diff
@@ -431,21 +425,22 @@ define wait
 endef
 
 define generate
-	$(call controller-gen,object:headerFile="hack/boilerplate.go.txt" paths="./...")
+	$(call controller-gen,object:headerFile="hack/boilerplate.gotxt" paths="./...")
```

Comment:
why header renamed?

Reply:
In order to add it to the rules in `.golic.yaml` file, adding go-style comments to all `*.txt` extensions would be too wide scope.